### ISSUE TITLE
Alien example is not working

### ISSUE THREAD

**Issue by sasha240100** [WhitestormJS/whitestorm.js#321]
- Animation is not playing
- Textures aren't working

###### Version:
 - [x] v2.x.x
 - [ ] v1.x.x

###### Issue type:
 - [x] Bug
 - [ ] Proposal/Enhancement
 - [ ] Question
 - [ ] Discussion

------

<details> 
<summary> <b>Tested on: </b> </summary>

###### Desktop
   - [ ] Chrome
   - [ ] Chrome Canary
   - [ ] Chrome dev-channel
   - [ ] Firefox
   - [ ] Opera
   - [ ] Microsoft IE
   - [ ] Microsoft Edge

###### Android
   - [ ] Chrome
   - [ ] Firefox
   - [ ] Opera

###### IOS
   - [ ] Chrome
   - [ ] Firefox
   - [ ] Opera

</details>

**Comment by hirako2000** [comment]
Broke due to `Importer.js` changes. [here](https://github.com/WhitestormJS/whs.js/commit/53aa5fec22c8e169c3939a3701b39819b5096ed9#diff-3dd230c717e2b6f187056e916b55e425).

I've just made a change to deal with instantiating the SkinnedMesh within Importer.js so that we don't have to override `parser` of the importer.
Removed custom material flag as it's not necessary anymore. but texture for some reason don't work.
